manhunt-base/src/main/kotlin/io/github/ytg1234/manhunt/base/ManhuntUtils.kt

```diff
@@ -45,30 +43,21 @@ val SERVER_QUESTION_PACKET_ID: Identifier = Identifier(MOD_ID, "question")
 @JvmField
 val CLIENT_ANSWER_PACKET_ID: Identifier = Identifier(MOD_ID, "answer")
 
-/**
- * Contains every [player][PlayerEntity] that has the mod
- * on their client.
- */
-@JvmField
-val haveMod: MutableList<PlayerEntity> = mutableListOf()
-
 /**
  * Manhunt's [Log4j logger][org.apache.logging.log4j.Logger]
  */
 @JvmField
 val LOGGER = LogManager.getLogger(MOD_NAME)!!
 
-/**
- * Contains the currently active speedrunner.
- */
-@JvmField
-var speedrunner: UUID? = null
-
 /**
  * The one and only Manhunt configuration.
  */
 @JvmField
-var CONFIG: ManhuntConfig? = null
+val CONFIG: ManhuntConfig = run {
```

Comment:
Would be better to make it lazy. Then it can't be a field but whatever.

Reply:
Making it lazy means making it nullable. It can't be.